specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-11-01/webapplicationfirewall.json

```diff
@@ -757,6 +757,36 @@
           "readOnly": true,
           "description": "Version of the managed rule set type."
         },
+        "managedRuleSetFeatures": {
```

Comment:
>managedRuleSetFeatures [](http://example.com/codeflow?start=9&length=22)

Probably rename as 'ruleSetFeatures' just to be consistent with the other properties.